### PR TITLE
ep01 round 1

### PR DESCRIPTION
**Pull request by CaitlinBate**

### CODE REVIEW

**Comment by CaitlinBate** [comment]
i see a formatting issue within the list under `Project Pages`

File: _episodes/01-plone.md [hunk 1]
```diff
@@ -3,48 +3,117 @@ title: Introduction to the Data Portal
...
+This page contains the following sections:
 
-### What is the Data Portal?
+1. TItle, citation, Node membership information and collaboration type (Tracker, Deployment or Data)
+2. An interactive map showing non-clickable tagging sites and clickable station-history information
+3. A list of the species studied by the project
+4. Contact information for Principal Investigators and Points of Contact
+5. Related URLs including:
```
Comment (naomitress) [inline _episodes/01-plone.md:32]: want to talk about the diffs when its a published project folder?

Comment (CaitlinBate) [reply _episodes/01-plone.md:32]: can do

File: _episodes/01-plone.md [hunk 2]
```diff
@@ -3,48 +3,117 @@ title: Introduction to the Data Portal
...
+If there are topics you think should be added, please reach out to OTNDC - this page can be updated at any time to remain valuable to users.
+
+## Statistics Page
+
+OTNDC has created a [Statistics](https://members.oceantrack.org/statistics) page on the Data Portal. These statistics are updated every 4 months during a Data Push, and can be used for presentations, exploration and general interest. Summaries include:
+
+- an interactive barplot showing the number of tagged individuals, per species, coloured by IUCN status
+- a pie chart showing the total number of detections in the OTN database, by type
+- an interactive chart showing the number of receivers "active" each month, coloured by year
+- an interactive map showing the "currently active" receivers we know are still in the water at the time of publishing. Only Node's whose data policies allow the publishing of station locations are included (not FACT, ACT or MigraMar)
```
Comment (naomitress) [inline _episodes/01-plone.md:118]: we can mention that other stats visuals can be custom created through a request to jon etc?

**Review by naomitress — APPROVED** [review]
2 comments, no blockers

**Pull request merged**